logging/logging/__init__.py

```diff
@@ -113,7 +113,15 @@ def basicConfig(level=INFO, filename=None, stream=None, format=None):
     root.addHandler(h)
```

Comment:
Commit message of this commit ("logging: Add minimal Handler base class."): please format it to ~78 chars lines.